server/chat-commands/core.ts

```diff
@@ -928,6 +928,7 @@ export const commands: ChatCommands = {
 	async showset(target, room, user, connection, cmd) {
 		this.checkChat();
 		const showAll = cmd === 'showteam';
+		const hideStats = target === 'hidestats';
```

Comment:
This should probably check the toID(target) rather than just the target.

Reply:
Also, what if you want to show just one set with stats hidden?

Reply:
I could make the arguments comma-delimited, I suppose and split on that. I don't think it'd be a common use case though; I'm picturing the use case for /showteam hidestats to be purely team-only for VGC purposes.

Reply:
Fair. I think it'd be nice to have, personally, but I'm not saying it has to be done.